coprocessor/fhevm-engine/host-listener/src/database/tfhe_event_propagate.rs

```diff
@@ -456,9 +469,18 @@ impl Database {
         &self,
         tx: &mut Transaction<'_>,
         event: &Log<AclContractEvents>,
+        transaction_hash: &Option<Handle>,
```

Comment:
event + transaction_hash + block_number could be a struct (like struct LogTfhe) shared or not (but less useful than for tfhe).